app/serializers/api/v2/serializable_course_enrichment.rb

```diff
@@ -0,0 +1,11 @@
+module API
+  module V2
+    class SerializableCourseEnrichment < JSONAPI::Serializable::Resource
+      type 'course_enrichments'
+
+      attributes :json_data, :status, :has_been_published_before?
```

Comment:
Should we really be exposing this as `json_data`? I think we should be hiding this particular implementation detail and exposing the attributes explicitly.

Reply:
Can we add tests to `spec/serializers/api/v2/serializable_course_spec.rb` to ensure that we serialize the expected attributes?

Reply:
The more I speak & think about it, it feels like the right thing to do here is to protect `manage-courses-frontend` from the enrichments as much as possible. I think we should explore exposing the course details in a 'published' form and a 'current' form (which would be the latest published or draft version).